src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -401,11 +400,11 @@ private void assertAndDump(String testDescription, boolean expected) {
                 dumpReport(testDescription, report);
```

Comment:
`testDescription` is not used in `dumpReport()` and can be removed.  Also, the JavaDoc for `dumpReport()` should  say `report` instead of `model`.  Sorry that I missed this on my first pass.

Reply:
Fixed